config/dev.js

```diff
@@ -46,4 +46,6 @@ module.exports = {
   extensionWorkshopUrl: 'https://extensionworkshop-dev.allizom.org',
 
   enableFeatureVPNPromo: true,
+
+  experiments: {'20210714_amo_vpn_promo': true},
```

Comment:
Don't we need the experiment in `config/default.js`?

Reply:
It will default to being disabled if it doesn't exist. This is the same approach we have taken in the past: https://github.com/mozilla/addons-frontend/commit/e026286240f653c8001fc90e64cb7fe287938bdb.

If you feel it's important to add it to `default.js` now I can, but it's not necessary.

Reply:
ok. No, it's probably fine but please format it like this:

```js
experiments: {
  '20210714_amo_vpn_promo': true,
}, 
```

😃